semantic-conventions/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Please update the changelog as part of any significant pull request.
 
 ## Unreleased
 
+- Inherit referenced attribute properties from `extend`
+  hierarchy before falling back to original attribute definition.
```

Comment:
"from hierarchy" does not tell much to me. Maybe:

```suggestion
- When an attribute is referenced using `ref:` from a group that already inherits the attribute with `extends:`, resolve the reference to the closest inherited attribute instead of the primary definition.
```